### PR TITLE
[NCL-533] Improve confirmation panels

### PR DESCRIPTION
**Pull request by thescouser89**
This is done by integrating modals with PNC.

out

### CODE REVIEW

**Comment by thescouser89** [comment]
Instead of using sweetalert library in #607, use Bootstrap modals instead as suggested.

**Comment by thescouser89** [comment]
@jdg-bot :raised_hands:

**Comment by alexcreasy** [comment]
@thescouser89 I'm loving it thanks for this, can I be a real pain and ask for two more slight changes ?

1) The confirm buttons should use the "danger" patternfly style, so it's red and people know they are doing something destructive? 

2) I feel cancel should maybe be the default button colour (i.e. grey) rather than using the "warning" orange colour, as cancel is the safe option.

The 4 different colour styles in patternfly have semantic meaning that we should try to abide by as it makes things more intuitive for the end user. The current method makes people think cancel is possibly dangerous and submit is perfectly safe.

WDYT?

File: ui/app/common/util/views/confirm-click.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="modal-header">
+    <h3 class="modal-title">{{ message }}</h3>
+</div>
+<div class="modal-footer">
+  <button class="btn btn-warning" type="button" ng-click="cancel()">Cancel</button>
```
Comment (alexcreasy) [inline ui/app/common/util/views/confirm-click.html]: Could the css class be: class="btn btn-default" ?

File: ui/app/common/util/views/confirm-click.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  <button class="btn btn-primary" type="button" ng-click="confirm()">Confirm</button>
```
Comment (alexcreasy) [inline ui/app/common/util/views/confirm-click.html]: could the css class be: class="btn btn-danger"

**Comment by alexcreasy** [comment]
Just to emphasise this is an awesome impl, in case it got lost in the critique above :+1:  :)

**Comment by thescouser89** [comment]
@alexcreasy Updated! Thanks for the suggestion :smiley:

**Comment by alexcreasy** [comment]
Test this please

**Comment by alexcreasy** [comment]
Perfect thanks @thescouser89 will merge when it passes CI!

**Comment by thescouser89** [comment]
@jdg-bot :star:

**Comment by alexcreasy** [comment]
I think I caused an extra build :worried:

**Pull request merged**